web/src/features/fireWeather/components/maps/WeatherMap.tsx

```diff
@@ -37,25 +38,20 @@ interface Props {
 }
 
 const WeatherMap = ({ redrawFlag }: Props) => {
-  const [wxStationsGeoJSON, setWxStationsGeoJSON] = useState({
-    type: 'FeatureCollection',
-    features: []
-  })
+  const dispatch = useDispatch()
+
+  const { stations } = useSelector(selectFireWeatherStations)
 
   useEffect(() => {
-    fetch(WEATHER_STATIONS_WEB_FEATURE_SERVICE)
-      .then(resp => resp.json())
-      .then(json => {
-        setWxStationsGeoJSON(json)
-      })
-  }, [])
+    dispatch(fetchWxStations())
```

Comment:
The station list is being requested (and thus downloaded and set twice). I'm not sure how worried I am about this.... since we're looking at having the stations on the map show all kinds of additional information we're probably going to end up having two api endpoints - one for a simple station list, and one for a list with extended information.

Reply:
Should I write a ticket for this?

Reply:
I think rather just add a note to https://app.zenhub.com/workspaces/wildfire-predictive-services-5e321393e038fba5bbe203b8/issues/bcgov/wps/849 ; we'll probably address it as part of that ticket.